Source/SwiftLintFramework/Extensions/Configuration+LintableFiles.swift

```diff
@@ -2,6 +2,13 @@ import Foundation
 import SourceKittenFramework
 
 extension Configuration {
+    public func lintableFiles(in files: [File], forceExclude: Bool) -> [File] {
+        return files.compactMap { file -> File? in
```

Comment:
Shouldn't this be

```swift
return files.flatMap { file in
    return file.path.map({ lintableFiles(inPath: $0, forceExclude: forceExclude) }) ?? []
}
```

so that you can pass directories to `--path` or `--paths` and have it still work?